### PR TITLE
change cl instructions for ssh key

### LINKED ISSUE #712: Clarify usage of command line interface to generate certificate signing requests
On Pantheon, adding SSL to your site is a completely self-serve option; Pantheon does not provide Private Keys or CSRs, or any SSH login for you to generate these. The key and certificates are cryptographically sensitive elements which you should take care to not send through email, as cleartext is very insecure.

Instead, we provide instructions for how to perform these steps yourself, and input into the form on the Domains/SSL tool in the site dashboard. The wizard you used generated the necessary command-line you’ll need to input into a Terminal:

```
openssl req -new -newkey rsa:2048 -nodes -out www_mysite_com.csr -keyout www_mysite_com.key -subj "/C=US/ST=California/L=San Francisco/O=MySite LLC/CN=www.mysite.com"
```

If you're using Mac OS X, you can open your Mac's Terminal application by going to Applications | Utilities | Terminal or using Spotlight to search. Sometimes this program is stored directly in Applications. Once you launch, you'll be presented with a window where you can directly paste that command.

Once you've generated the CSR and Key, you can find the files to provide to your certificate provider.

After receiving your signed certificate and any intermediary certificates you'll need to paste the contents in the SSL tool on the Pantheon dashboard.

### PR DESCRIPTION
**Pull request by nataliejeremy**
closes #712 This PR updates the CL instructions for generating a key.

### CODE REVIEW

**Comment by nataliejeremy** [comment]
I'll also be removing the screenshots for downloading and installing git; keeping the Pantheon screenshot.

**Comment by nataliejeremy** [comment]
@bmackinney and @suzannealdrich Please review--thanks!

**Comment by nataliejeremy** [comment]
@bmackinney or @ari-gold Can you review? Not sure Suzanne will come back to this (she submitted the issue).

File: source/docs/articles/users/generating-ssh-keys.md [hunk 1]
```diff
@@ -5,80 +5,78 @@ category:
   - getting-started
 keywords: ssh keys, ssh, generate keys  
 ---
-SSH as a protocol is not supported on Pantheon. <!--You can not connect via SSH using Putty.--> These directions are to allow you have passwordless access if you configure Drush or SFTP to use the keys setup by putty.
+On Pantheon, adding SSL to your site is a completely self-serve option; Pantheon does not provide Private Keys or CSRs, or any SSH login for you to generate these. The key and certificates are cryptographically sensitive elements that you should not send through email, as cleartext is very insecure.
```
Comment (ari-gold) [inline source/docs/articles/users/generating-ssh-keys.md]: Why open a doc about SSH keys with a reference to SSL?

**Comment by nataliejeremy** [comment]
Closing per review from Ari.

File: source/docs/articles/users/generating-ssh-keys.md (hunk 1 above)
Comment (suzannealdrich) [inline source/docs/articles/users/generating-ssh-keys.md]: Agreed. That line should go in the adding SSL certs for HTTPS article.

**Pull request closed without merging**